scripts/test.bats

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bats
+
+# SPDX-FileCopyrightText: 2021 Arthur Breitman
+# SPDX-License-Identifier: LicenseRef-MIT-Arthur-Breitman
+
+ligo_exe="baseDAO-ligo-meta"
+
+@test "Print LIGO metadata" {
+  "$ligo_exe" print-metadata --frozen-token-symbol=POG > metadata.json
```

Comment:
this command would fail right now and I'm not sure we'll ever need this script here (we'll likely use the Makefile).

I'd remove this file entirely TBH

Reply:
Removed this.